### ISSUE TITLE
Add AARCH64 demo

### ISSUE THREAD

**Issue by redfast00** [AlexAltea/unicorn.js#15]
It would be pretty neat if there was an AARCH64 demo as well.

**Comment by AlexAltea** [comment]
Agreed! I wanted to keep the demo simple and it only supports integers up to 32 bits even if Unicorn.js supports 64 bit registers as well, so this would require rather major (though trivial) changes for which I have not enough time for at the moment.

I'll keep the issue open, just in case me (or someone else) is motivated enough to do it in the future. :-)

**Comment by AlexAltea** [comment]
Done at https://github.com/AlexAltea/unicorn.js/commit/ca855f4c4f3a5e0a5d88dd02bb79be03e56925cf based on code written by @bitterbit.
